lib/sparse/src/index/inverted_index/inverted_index_mmap.rs

```diff
@@ -196,8 +195,7 @@ impl InvertedIndexMmap {
         let file_header: InvertedIndexFileHeader = read_json(&config_file_path)?;
         // read index data into mmap
         let file_path = Self::index_file_path(path.as_ref());
-        let mmap = open_read_mmap(file_path.as_ref())?;
-        madvise::madvise(&mmap, madvise::Advice::Normal)?;
+        let mmap = open_read_mmap(file_path.as_ref(), Advice::Normal.into())?;
```

Comment:
For the sake of better code navigation, let's use something like this:

```suggestion
        let mmap = open_read_mmap(file_path.as_ref(), AdviceSetting::from(Advice::Normal))?;
```